src/NServiceBus.Transport.Msmq.AcceptanceTests/TimeToBeReceived/When_sending_message_with_ttbr.cs

```diff
@@ -75,7 +75,11 @@ public async Task Suppress_native_ttbr_if_enabled_inside_transaction()
         {
             var context = await Scenario.Define<Context>()
                 .WithEndpoint<Sender>(endpoint => endpoint
-                    .CustomConfig(endpointConfiguration => endpointConfiguration.UseTransport<MsmqTransport>().DisableNativeTimeToBeReceivedInTransactions())
+                    .CustomConfig(endpointConfiguration =>
+                    {
+                        var transportSettings = (MsmqTransport)endpointConfiguration.ConfigureTransport();
+                        transportSettings.DisableNativeTtbrInTransactions = true;
```

Comment:
Should this be renamed to `DisableNativeTTBRInTransactions` ?

Reply:
might make more sense. The style-checks will complain about that I think but we can ignore that I guess

Reply:
I renamed this to `UseNonNativeTimeToBeReceivedInTransactions` to be more clear about what this setting does. Is that ok for you too @andreasohlund ?